### PR TITLE
Improve ShaderProgram construction

### PR DESCRIPTION
**Pull request by Qard**
I've changed from using `cat` to using the `read_file` macro function, which _should_ hopefully allow this to work on Windows someday. 🤔

### CODE REVIEW

File: src/shader_program.cr [hunk 1]
```diff
@@ -4,16 +4,15 @@ class Glove::ShaderProgram
     @uniform_locations = {} of String => Int32
   end
 
-  def self.from(vertex_shader_filename : String, fragment_shader_filename : String)
-    from(
-      File.read(vertex_shader_filename),
-      File.read(fragment_shader_filename)
-    )
+  def self.from(vertex : Path, fragment : Path)
```
Comment (denisdefreyne) [inline src/shader_program.cr]: I generally prefer longer, more-descriptive variable names, but I’m OK with either.

**Review by denisdefreyne — APPROVED** [review]

**Comment by denisdefreyne** [comment]
Feel free to merge — the one comment I left is not a blocker.

**Comment by Qard** [comment]
I'll fix it first. Probably later tonight, just need to find some time to update it. 👍

File: src/shader_program.cr (hunk 1 above)
Comment (Qard) [reply src/shader_program.cr]: fixed

**Pull request merged**